tsfile/src/main/java/org/apache/iotdb/tsfile/file/metadata/ChunkMetaData.java

```diff
@@ -49,11 +49,15 @@
 
   private TSDataType tsDataType;
 
+  /** version is set  according to its belonging ChunkGroup only when being queried, so it is not

```

Comment:
/**
code
*/
Is this format better?

Reply:
The comment for version variable is not detailed.

Reply:
Fixed.